source/globals.d.ts

```diff
@@ -1,14 +1,15 @@
-declare const browser: AnyObject;
-
-type AnyObject = {[key: string]: any};
+// TODO: Type anything that is of type AnyObject
+// for browser, we can cherry pick from the chrome namespace from @types/chrome
+// ensuring that functions return Promises where applicable.
+type AnyObject = Record<string, any>;
 
 declare module 'select-dom' {
 	import 'select-dom';
 
-	function select(selector: string, parent?: any): any | null;
+	function select<T extends HTMLElement = HTMLElement>(selector: string, parent?: string|Element|DocumentFragment): T | null;
```

Comment:
@bfred-it, the `parent` argument in all these functions in your types in your package will need to support `DocumentFragment` as well. I think you mentioned this already, but I'll need a package with that published/pre-release. For now on my local I've modified the index.d.ts file to get things working.

Reply:
Ahh yeah I added it to `select.all` but not `select`. Will do